app/src/main/kotlin/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -767,7 +777,7 @@ class WebViewActivity :
                                         }
                                     sendExternalBusMessage(
                                         ExternalConfigResponse(
-                                            id = JSONObject(message).get("id"),
+                                            id = message.toJsonObjectOrNull()?.getIntOrNull("id") ?: 0,
```

Comment:
```suggestion
                                            id = json.getIntOrElse("id", 0),
```

Double check but I think this should do the same.